recOrder/io/utils.py

```diff
@@ -165,3 +167,16 @@ def get_unimodal_threshold(input_image):
             max_dist = per_dist
     assert best_threshold > -np.inf, 'Error in unimodal thresholding'
     return best_threshold
+
+def ram_message():
+    BYTES_PER_GB = 2**30
+    gb_available = psutil.virtual_memory().total / BYTES_PER_GB
+    if gb_available < 32:
+        return ' \n'.join(textwrap.wrap(
+               f'\033[91mWARNING:\033[0m recOrder reconstructions often require more than the {gb_available:.1f} ' \
```

Comment:
I'm not sure if we ever use `logging` in the CLI. If we do maybe it would make sense to not manually format the message?

Reply:
Also ANSI code support on Windows seems to be version-specific.

Reply:
Good call on `print` vs. `logging`. I just changed the online mode `print`s to `logging.info`. 

Re: ANSI code support: I tested that it works in powershell, but you might be right with cmd.exe. I think the worst that will happen is that you'll see "\033[91mWARNING:\033[0m", which isn't the end of the world, so I'm not too concerned.

Reply:
We will see something like `[Warning] WARNING ...` in online mode, but that's only cosmetic.

Reply:
I did not think it through, we are now actually seeing:
```
10:21:26 INFO WARNING: recOrder reconstructions often require more than the
16.0 GB of RAM that this computer is equipped with. We recommend
starting with reconstructions of small volumes ~1000 x 1000 x 10 and
working up to larger volumes while monitoring your RAM usage with Task
Manager or htop.
```
And this message will only be displayed in console/log, without flashing a warning in the GUI.
Maybe we should have it do `show_warning(ram_message())` instead.

Reply:
Addressed in #174